src/ui/components/row.js

```diff
@@ -0,0 +1,51 @@
+const log4js = require("log4js");
```

Comment:
This whole file is not used anymore, or?
(Can't verify it properly on my phone 😅)